bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/CameraCapability.java

```diff
@@ -0,0 +1,108 @@
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.handler.capability;
+
+import static org.openhab.binding.netatmo.internal.NetatmoBindingConstants.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.netatmo.internal.api.dto.NAHomeDataPerson;
+import org.openhab.binding.netatmo.internal.api.dto.NAHomeEvent;
+import org.openhab.binding.netatmo.internal.api.dto.NAHomeStatusModule;
+import org.openhab.binding.netatmo.internal.api.dto.NAObject;
+import org.openhab.binding.netatmo.internal.deserialization.NAObjectMap;
+import org.openhab.binding.netatmo.internal.handler.NACommonInterface;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.CameraChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.ChannelHelper;
+import org.openhab.binding.netatmo.internal.providers.NetatmoDescriptionProvider;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.StateOption;
+
+/**
+ * {@link CameraCapability} give to handle Welcome Camera specifics
+ *
+ * @author Gaël L'hopital - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class CameraCapability extends Capability {
+    private final CameraChannelHelper cameraHelper;
+    protected @Nullable String localUrl;
+    private final NetatmoDescriptionProvider descriptionProvider;
+    private final ChannelUID personChannelUID;
+
+    public CameraCapability(NACommonInterface handler, NetatmoDescriptionProvider descriptionProvider,
+            List<ChannelHelper> channelHelpers) {
+        super(handler);
+        this.descriptionProvider = descriptionProvider;
+        this.personChannelUID = new ChannelUID(thing.getUID(), GROUP_LAST_EVENT, CHANNEL_EVENT_PERSON_ID);
+        this.cameraHelper = (CameraChannelHelper) channelHelpers.stream().filter(c -> c instanceof CameraChannelHelper)
+                .findFirst().orElseThrow(() -> new IllegalArgumentException(
+                        "CameraHandler must have a CameraChannelHelper, file a bug."));
+    }
+
+    @Override
+    public void updateHomeStatusModule(NAHomeStatusModule newData) {
+        super.updateHomeStatusModule(newData);
+        String vpnUrl = newData.getVpnUrl();
+        boolean isLocal = newData.isLocal();
+        // TODO : j'ai un problème de séquence d'exécution ici. Le channel helper est appelé avant
+        // que les valeurs de vpnUrl et localUrl lui soient fournies, donc Live Snapshot et LiveStream url sont undef au
+        // début.
```

Comment:
Another TODO in French. :) Remember to remove or translate.

Reply:
Corrected